emulator/blockchain.go

```diff
@@ -888,9 +889,13 @@ func (b *Blockchain) GetTransactionResult(txID flowgo.Identifier) (*access.Trans
 		return nil, err
 	}
 
+	statusCode := codes.OK
+	if storedResult.ErrorCode > 0 {
+		statusCode = codes.Canceled
+	}
```

Comment:
The code that sets the status code is here:
https://github.com/onflow/flow-go/blob/37ece4cd39961442ebae8d622b63fb5a8cf8da18/engine/execution/rpc/engine.go#L306

```suggestion
	statusCode := 0
	if storedResult.ErrorCode > 0 {
		statusCode = 1
	}
```

Reply:
Ah it is not grpc code, thanks